cmd/tailscale/cli/serve.go

```diff
@@ -709,3 +709,19 @@ func (e *serveEnv) runServeFunnel(ctx context.Context, args []string) error {
 	}
 	return nil
 }
+
+// checkHasAccess checks three things: 1) an invite was used to join the
+// Funnel alpha; 2) HTTPS is enabled; 3) the node has the "funnel" attribute.
+// If any of these are false, an error is returned describing the problem.
+func checkHasAccess(capabilities []string) error {
```

Comment:
rename capabilities to nodeAttrs ?

and describe it in the godoc for this func a bit

Reply:
but LGTM still otherwise